README.md

```diff
@@ -72,7 +72,12 @@ reasonably POSIX-compliant system.
 Installation
 ------------
 
-For some Linux distributives deb and rpm packages are avalillable.
+### Package installation
+
+For some Linux distributives internal packages with `sqlite_fdw` are avalillable.
```

Comment:
Typo: avalillable.
I think it should be "available".

Reply:
Fixed

Reply:
Thanks. Confirmed.